presto-main/src/main/java/io/prestosql/sql/planner/LogicalPlanner.java

```diff
@@ -341,25 +368,29 @@ private RelationPlan createTableCreationPlan(Analysis analysis, Query query)
                 statisticsMetadata);
     }
 
-    private RelationPlan createInsertPlan(Analysis analysis, Insert insertStatement)
+    private RelationPlan getInsertPlan(Analysis analysis,
```

Comment:
First argument should go on the next line

Reply:
done